package.json

```diff
@@ -28,7 +28,7 @@
     "grunt-contrib-watch": "~0.5.3",
     "grunt-eslint": "^17.3.1",
     "grunt-postcss": "^0.8.0",
-    "intern": "2.2.0",
+    "intern": "^2.2.2",
```

Comment:
this may be unnecessary. I tried the upgrade in response to an intern error message but it may have been resolved by a missing Selenium browser config in intern.js. I can revert to the original.